src/.eslintrc.json

```diff
@@ -52,7 +52,10 @@
     "import/no-unresolved": ["off"],
     "@typescript-eslint/no-unused-vars": [
       "error",
-      { "ignoreRestSiblings": true }
+      {
+        "argsIgnorePattern": "^_action$",
```

Comment:
Nice! Didn't know about that option. I've heard it's common to allow unused arguments when prefixed with an underscore, and in fact WebStorm (with the built-in inspection that looks for unused arguments) did silence its warning when I prefixed with an underscore - but I added the ignore comment just for the remaining eslint warning.

(Btw, I wrote this comment directly in the linked commit, but seems like it got posted to the void. I'll try again here!)